src/wallet/walletdb.h

```diff
@@ -49,7 +49,7 @@ enum class DBErrors
     TOO_NEW,
     LOAD_FAIL,
     NEED_REWRITE,
-    RESCAN_REQUIRED
+    NEED_RESCAN
```

Comment:
if you have to retouch
```suggestion
    NEED_RESCAN,
```